### PR TITLE
Card components

### PR DESCRIPTION
**Pull request by darshandamre**
Added card components:
- vertical and horizontal cards
- with badges
- with dismiss button
- with text overlay
- text only cards
Preview [component-lib-git-cards-darshandamre.vercel.app/pages/docs.html#card](https://component-lib-git-cards-darshandamre.vercel.app/pages/docs.html#card)

### CODE REVIEW

File: components/button.css [hunk 1]
```diff
@@ -4,6 +4,13 @@
   border-radius: 5px;
   cursor: pointer;
   font-weight: 600;
+  background: #3b3b3b;
```
Comment (gopalbbharadva) [inline components/button.css]: Suggestion: Use variables instead of hard code string.(Maybe you forgot to do it several places.)

**Review by gopalbbharadva — APPROVED** [review]

File: components/cards.css [hunk 2]
```diff
@@ -0,0 +1,135 @@
+.card {
+  max-width: 18.75rem;
+  background-color: #171717;
+  border-radius: 5px;
+  text-align: center;
+
```
Comment (tushargoyal812) [inline components/cards.css:6]: remove unnecessary space

File: components/cards.css [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+}
+
+.card--horizontal .card__image {
+  max-height: 24.375rem;
+}
+
+.card--horizontal .card__content {
+  margin: 2.5rem;
+  flex-grow: 1;
+
```
Comment (tushargoyal812) [inline components/cards.css:86]: also here

**Review by tushargoyal812 — COMMENTED** [review]
i think everything looks good.

File: components/button.css (hunk 1 above)
Comment (darshandamre) [reply components/button.css]: yes will do that, at this point i am running out of variable names

File: components/cards.css (hunk 2 above)
Comment (darshandamre) [reply components/cards.css:6]: thank you for pointing this out, this was intentional by me to kind of highlight the position value of this css class

File: components/cards.css (hunk 3 above)
Comment (darshandamre) [reply components/cards.css:86]: this class is a flex children as well as flex container, so i've separated the flex children and flex container properties with an empty line

Comment (darshandamre) [reply components/cards.css:86]: appreciate your review

**Pull request merged**